lua/starfall/libs_cl/particle_effect.lua

```diff
@@ -254,9 +235,9 @@ function particleef_methods:setUpVector(id,value)
 end
 
 
---- Sets the forward direction for given control point.
--- @param number Child Control Point ID (0-63)
--- @param number Parent
+--- Sets the parent for given control point.
+-- @param number id Child Control Point ID (0-63)
+-- @param number parentid Parent
```

Comment:
Looks like the param name here doesn't match

Reply:
I don't remember if that causes issues or not. probably not

Reply:
No, it doesn't, I should've changed the argument in code as well but I think here I was still trying not to change any code